uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java

```diff
@@ -167,5 +167,37 @@ public void testAddColumnBeforeActions(){
         assertEquals("actions" , columnPicker.getDataGrid().getColumn(4).getDataStoreName());
 
     }
+
+    @Test
+    public void testAddColumnBeforeActionsOnAddColumn(){
+        UberfireColumnPicker<String> columnPicker = new UberfireColumnPicker<>(new DataGrid<>());
+        ColumnMeta<String> name = createColumnMeta("Name","name", true,true);
+        ColumnMeta<String> age = createColumnMeta("Age","age", true,true);
+        ColumnMeta<String> description = createColumnMeta("Description","description", false,true);
+        ColumnMeta<String> actions = createColumnMeta("Actions","actions", true,false);
+        ColumnMeta<String> etc = createColumnMeta("Etc","etc", true,true);
+
+        columnPicker.addColumn(name);
+        columnPicker.addColumn(age);
+        columnPicker.addColumn(description);
+        columnPicker.addColumn(actions);
+        columnPicker.addColumn(etc);
+
+        assertEquals(4, columnPicker.getDataGrid().getColumnCount());
+        assertEquals("name", columnPicker.getDataGrid().getColumn(0).getDataStoreName());
+        assertEquals("age", columnPicker.getDataGrid().getColumn(1).getDataStoreName());
+        assertEquals("actions", columnPicker.getDataGrid().getColumn(3).getDataStoreName());
```

Comment:
can you just order the assert in the order it is expected the columns to show? That would make it easier to read.

Reply:
ok